cmd/logger.go

```diff
@@ -137,6 +155,24 @@ func getSource() string {
 	return fmt.Sprintf("[%s:%d:%s()]", filename, lineNum, funcName)
 }
 
+func printJSON(level, source, cause, msg string, stack []string, data ...interface{}) {
+	log := &logEntry{
+		Level:   level,
+		Source:  source,
+		Stack:   stack,
+		Cause:   cause,
+		Time:    time.Now().UTC().Format(time.RFC3339Nano),
+		Message: fmt.Sprintf(msg, data...),
+	}
+
+	logJSON, _ := json.Marshal(log)
```

Comment:
If there is an error Marshalling we can `panic(err)` i think. Rather than printing an empty message, we are expecting all our values to be JSON friendly.